src/main/java/Event.java

```diff
@@ -0,0 +1,27 @@
+public class Event extends Task {
+    private String from;
+    private String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    public Event(String input) {
+        super(input.substring(input.indexOf(" ") + 1, input.indexOf("/from") - 1));
```

Comment:
Perhaps a better way is to use constants instead of these numbers so that you can make modifications easier later on